tests/jerry/es.next/object-prototype-fromEntries.js

```diff
@@ -0,0 +1,143 @@
+// Copyright JS Foundation and other contributors, http://js.foundation
```

Comment:
The name of the testcase is incorrect, since it's **not** an object prototype routine.